test/auth_cluster/test_auth_no_quorum.py

```diff
@@ -54,3 +55,44 @@ async def test_auth_no_quorum(manager: ManagerClient) -> None:
             auth_provider=PlainTextAuthProvider(username=role, password=role))
     names = [row.role for row in cql.execute(f"LIST ROLES", execution_profile='whitelist')]
     assert(set(names) == set(['cassandra'] + roles))
+
+
+async def trigger_snapshot(manager: ManagerClient, server: ServerInfo) -> None:
+    cql = manager.get_cql()
+    host = cql.cluster.metadata.get_host(server.ip_addr)
+    group0_id = (await cql.run_async(
+        "select value from system.scylla_local where key = 'raft_group0_id'",
+        host=host))[0].value
+    await manager.api.client.post(f"/raft/trigger_snapshot/{group0_id}", host=server.ip_addr)
+
+"""
+Tests raft snapshot transfer of auth data.
+"""
+@pytest.mark.asyncio
+async def test_auth_raft_snapshot_transfer(manager: ManagerClient) -> None:
+    servers = await manager.servers_add(3)
```

Comment:
I think the test should even work with one-node cluster in the beginning. You mentioned in another comment that the snapshot is not transfered unless you have at least 3 nodes. I don't think this should affect anything. Did you try to investigate why a snapshot is not transfered?

Reply:
I confirm it works with servers_add(1), switching